src/components/MaterialUI/DataDisplay/Address.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles";
+import { TextProps } from "../textProps";
+import TooltippedTypography from "./TooltippedElements/TooltippedTypography";
+import _ from "lodash";
+
+const useStyles = makeStyles(theme => ({}));
+
+const Address = ({
+	name,
+	contactPerson,
+	line1,
+	line2,
+	city,
+	province,
```

Comment:
"province" is maybe too much Canada-Oriented as it may have different names in other countries. In our database, we call that "regionName", it would be more a generic term.